RawSpeed/DngDecoderSlices.cpp

```diff
@@ -160,6 +313,27 @@ void DngDecoderSlices::decodeSlice(DngDecoderThread* t) {
         mRaw->setError(err.what());
       }
     }
+    /* Deflate compression */
+  } else if (compression == 8) {
+    unsigned char *uBuffer = NULL;
+    float *floatBuffer = NULL;
+    while (!t->slices.empty()) {
+      DngSliceElement e = t->slices.front();
+      if (!uBuffer)
+        uBuffer = new unsigned char [e.width * e.height * sizeof(float)];
```

Comment:
At least in C, `typeof(sizeof())` is `size_t`, but are `width` and `height` of `size_t` type too?
If not, the multiplication can somewhat easily overflow, i believe.